specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-10-01-preview/RoleManagementPolicy.json

```diff
@@ -558,9 +558,9 @@
           "type": "string",
           "description": "The notification level.",
           "enum": [
-            "NONE",
-            "CRITICAL",
-            "ALL"
+            "None",
```

Comment:
as per [https://aka.ms/bcforapi](https://aka.ms/bcforapi) - this is a breaking change

Reply:
Our API was always using Pascal case, just the spec had all caps incorrectly.